resources/leiningen/new/dw_code_exercise/submit.clj

```diff
@@ -0,0 +1,26 @@
+(ns {{name}}.submit
+  (:require [clojure.java.shell :refer [sh] :as shell]
+            [clojure.string :as str])
+  (:gen-class))
+
+(defn zip-name []
+  (-> (sh "pwd")
+      :out
+      (str/split #"/")
+      last
+      str/trim
+      (str ".zip")))
```

Comment:
The zip's name can just be `"{{name}}.zip"`, right? No need to shell out to `pwd` and manipulate the output.

Reply:
Oh, yeah, of course.